bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -48,6 +48,10 @@
   {% endcall %}
 {% endblock %}
 
+{% block string_data %}
+data-get-firefox="{{ _('Get Firefox') }}"
```

Comment:
Could this be either `"Try it now"` or `"Get it now"`? https://bugzilla.mozilla.org/show_bug.cgi?id=1164424#c3